app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
You should call chaining keys in dicts only in cases when you are sure in its structure. In production cases, it is safer to use `get("smth1", {}).get("smth2")` structure.